server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/GetMessagesMethodStepdefs.java

```diff
@@ -677,6 +677,12 @@ public void assertKeywordsOfMessageShouldDisplay(List<String> keywords) throws E
             .containsOnlyElementsOf(keywords);
     }
 
+    @Then("^the empty keywords of the message$")
```

Comment:
Maybe it's me who is really tired but I thunk "the message has no keywords would be a better naming"

Reply:
I changed it